pkg/api-server/inspect_endpoints_test.go

```diff
@@ -60,6 +60,22 @@ func serviceSelector(name, protocol string) *mesh_proto.Selector {
 	}
 }
 
+func defaultTrafficRoute(mesh string) *core_mesh.TrafficRouteResource {
```

Comment:
nitpick: could've been a reusable `route-all-default` test resource - but we're not going to develop old policies more so I think this can stay.